### PR TITLE
Pr/php8

### PR DESCRIPTION
**Pull request by fritz-gerneth**

### CODE REVIEW

**Comment by fritz-gerneth** [comment]
@sandrokeil I think you need to release a new version of the updated `php-cs-fixer-config` first

**Comment by prolic** [comment]
see https://github.com/prooph/php-cs-fixer-config/releases/tag/v0.4.0

**Comment by fritz-gerneth** [comment]
Quite a few changes actually:
- Update License headers (though we can update them soon again anyway)
- Adapted PHPUnit configuration for version 9 
- Extra flag for Xdebug3, travis is using it already
- Moved PHP-CS as an extra step to travis instead of executing it on every target. Safes a bit of extra work, but also solves the PHP8-issue with PHP-CS

**Comment by fritz-gerneth** [comment]
I think this is ready now @prolic @codeliner

File: .travis.yml [hunk 1]
```diff
@@ -3,20 +3,33 @@ language: php
 matrix:
   fast_finish: true
   include:
-    - php: 7.1
+    - php: 7.3
       env:
         - DEPENDENCIES=""
+        - XDEBUG_MODE="coverage"
         - TEST_COVERAGE=true
```
Comment (prolic) [inline .travis.yml:10]: this is not used anymore?

File: .travis.yml [hunk 2]
```diff
@@ -3,20 +3,33 @@ language: php
...
-    - php: 7.1
+    - php: 7.3
+      env:
+          - DEPENDENCIES=""
+          - PHP_CS=true
```
Comment (prolic) [inline .travis.yml:14]: this is not used anymore?

Comment (fritz-gerneth) [reply .travis.yml:14]: It is ( https://travis-ci.org/github/prooph/common/jobs/752820990 ) , CS is only executed once though now, not on every PHP Version.

https://xdebug.org/docs/upgrade_guide for reference about XDEBUG_MODE in xdebug 3

**Review by prolic — APPROVED** [review]

**Pull request merged**